app/views/maps/index.html.erb

```diff
@@ -1,41 +1,44 @@
 <div class="container">
-  <% ab_finished('viewed_maps', reset: false) %>
     <div class="row">
-        <div class="col-lg-12">
-            <h2 class="page-header underline">EBWiki below maps <%= @cases.count %> cases of violence against people of color. Read more <%= link_to 'about us',about_path %></h2>
-            <% if user_signed_in? %>
-	            Click the follow button below the image on any case page and we will keep you updated as these cases proceed.</p>
-						<% else %>
-	            <%= link_to 'Login', new_user_registration_path %> and follow specific cases and we will update you as they proceed.
-						<% end %>
-						<div class="text-center">
-							<h3>
-								<%= link_to 'Back to the Main Page', root_path, class: "btn btn-warning" %>
-								<%= link_to 'Get our newsletter', "http://eepurl.com/bt5QCz", class: "btn btn-default" %>
-						  </h3>
-						</div>
-						<h2 class="page-subheader">Click on the map pins below to learn more.</h2>
-        </div>
-    </div>
-
-    <!-- Map -->
-    <div class="row">
-		<div class="col-lg-12 articleMap">
-		   	<div id="markers-on-the-map">
-				<div class="page-header">
-			      	<h1>Map at a specified location</h1>
-			        <p>Display a map at a specified location and zoom level</p>
-			    </div>
-    			<p>This example displays a movable map initially centered on the <b>Brandenburg Gate</b> in the centre of Berlin <i>(52.5159°N, 13.3777°E)</i></p>
-    			<div id="map"></div>
-    			<h3>Code</h3>
-			    <p>
-			        The <code>map.setCenter()</code> method and <code>map.setZoom() </code>method are able to control the location of the map.<br>
-			    </p>
-			   
-			  </div>
-		  </div>
-	    </div>
+    	<div class="col-lg-12">
+	     	<h2 class="page-header underline">EBWiki below maps <%= @cases.count %> cases of violence against people of color. Read more <%= link_to 'about us',about_path %></h2>
+	        <% if user_signed_in? %>
+		     	Click the follow button below the image on any case on the map. We will keep you updated as these cases proceed.</p>
+			<% else %>
+		    	<%= link_to 'Login', new_user_registration_path %> and follow specific cases and we will update you as they proceed.
+			<% end %>
+			<div class="text-center">
+				<h3>
+					<%= link_to 'Back to the Main Page', root_path, class: "btn btn-warning" %>
+					<%= link_to 'Get our newsletter', "http://eepurl.com/bt5QCz", class: "btn btn-default" %>
+				</h3>
+			</div>
+			<h2 class="page-subheader">Click on the map pins below to learn more.</h2>
+			<div id="markers-on-the-map">
+			    <p>This example displays a movable map initially centered on the <b>United States</b> in the centre of Berlin <i>(52.5159°N, 13.3777°E)</i></p>
+	    		<div id="map page-header">
+		    		<div style="width: 640px; height: 480px; margin: auto" id="mapContainer"></div>
+					<!-- HERE Maps -->
+					<script>
```

Comment:
Are you able to write this in a separate JS file in the asset pipeline and reference it normally?  Or does that run into an issue with Cloud 9?

Reply:
I can. I initially wanted to do this the simplest way possible, and the next step with this would be to separate this into different files

Reply:
I'd prefer that we go ahead and just do that now, unless the issues with Cloud 9 prevent that.

Reply:
I don't think there are Cloud9 issues that prevent that. Soon come...

Reply:
This is done.